joomla/joomla-cms
# [6.2] Enable automatic title for admin modules

Pull Request for Issue https://github.com/joomla/joomla-cms/issues/35587 (partly)
Related to https://github.com/joomla/joomla-cms/pull/21330

### Summary of Changes
Make sure automatic title is enabled by default like we had it in 2.5


### Testing Instructions
[Install package build for this PR by dron](https://ci.joomla.org/artifacts/joomla/joomla-cms/4.2-dev/36954/downloads/55066/Joomla_4.2.0-beta1-dev+pr.36954-Development-Full_Package.zip) and confirm that the setting `Automatic Title` is enabled for `Popular Articles`, `Recently Added Articles`,  `Logged-in Users` and `Latest Actions`.


### Actual result BEFORE applying this Pull Request
The setting is disabled and just the original title is displayed.


### Expected result AFTER applying this Pull Request
The setting is enabled and shows the generated title instead of the default.


### Documentation Changes Required
